packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.utils.ts

```diff
@@ -3,6 +3,7 @@ import isNumber from 'lodash/isNumber';
 import { ConceptMetadata } from '@openmrs/esm-patient-common-lib';
 
 export function calculateBMI(weight: number, height: number): number {
+  if (!weight && !height) return;
```

Comment:
Should this be `||` instead? 

Also should this function have a default return value?

Reply:
Yeah, it should. I don't think it should have a default return value.